addons/l10n_pk/data/account_tax_wh_report.xml

```diff
@@ -0,0 +1,402 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_pk_wh_vat_form" model="account.report">
+        <field name="name">Withholding Tax Report - Pakistan (PK)</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.pk"/>
+        <field name="filter_fiscal_position" eval="True"/>
+        <field name="availability_condition">country</field>
+        <field name="column_ids">
+            <record id="l10n_pk_wh_vat_column_net" model="account.report.column">
+                <field name="name">Balance</field>
+                <field name="expression_label">balance</field>
+            </record>
+            <record id="l10n_pk_wh_vat_column_tax" model="account.report.column">
+                <field name="name">Tax Amount</field>
+                <field name="expression_label">taxbalance</field>
```

Comment:
```suggestion
                <field name="expression_label">tax</field>
```
just a suggestion here too (plus my IDE complains it's not proper English :smile: )

Reply:
Done